data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/beaconhandlers/BeaconStateRootHandler.java

```diff
@@ -59,21 +59,21 @@ public BeaconStateRootHandler(final ChainDataProvider provider, final JsonProvid
   @OpenApi(
       path = ROUTE,
       method = HttpMethod.GET,
-      summary = "Get the beacon chain state root that matches the specified slot.",
+      summary = "Get the beacon chain state root for the specified slot.",
       tags = {TAG_BEACON},
       queryParams = {
-        @OpenApiParam(
-            name = SLOT,
-            description = "Slot to query in the canonical chain (head or ancestor of the head).")
+        @OpenApiParam(name = SLOT, description = "Slot to query in the canonical chain.")
       },
-      description = "Returns beacon chain state root that matches the specified slot.",
+      description = "Returns the beacon chain state root for the specified slot.",
       responses = {
-        @OpenApiResponse(status = RES_OK, content = @OpenApiContent(from = Bytes32.class)),
+        @OpenApiResponse(
+            status = RES_OK,
+            content = @OpenApiContent(from = Bytes32.class),
+            description = "The beacon chain's `state_root`(`Bytes32`) for the specified slot."),
```

Comment:
elsewhere we have "the beacon chain state root" without apostrophe

Reply:
Yep, I prefer without the apostrophe. Will fix.